features/networking/egress-ip.feature

```diff
@@ -136,18 +135,30 @@ Feature: Egress IP related features
     # Patch egress IP to the project twice
     Given I have a project
     And as admin I successfully merge patch resource "netnamespace/<%= project.name %>" with:
-      | {"egressIPs": ["<%= cb.newip %>"]} |
-    And as admin I successfully merge patch resource "netnamespace/<%= project.name %>" with:
-      | {"egressIPs": ["<%= cb.valid_ip %>"]} |
+      | {"egressIPs": ["<%= cb.valid_ips[0]%>"]} |
+
+    And I wait up to 30 seconds for the steps to pass:
+    """
+    When I run command on the "<%= node.name%>" node's sdn pod:
+      | bash | -c | ip address show <%= cb.interface %> |
```

Comment:
all egress IPs have the `:eip` suffix label on the device?  And they all have `secondary` flag?

```
      2: ens5: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 9001 qdisc mq state UP group default qlen 1000
          link/ether 0e:82:96:67:cf:6c brd ff:ff:ff:ff:ff:ff
          inet 10.0.222.222/19 brd 10.0.223.255 scope global dynamic noprefixroute ens5
             valid_lft 3428sec preferred_lft 3428sec
          inet 10.0.214.169/19 brd 10.0.223.255 scope global secondary ens5:eip
             valid_lft forever preferred_lft forever
```

It looks like we can lookup by label
```
ip a show label ens5:eip
```

It is also possible to lookup all addrs with the `secondary` flag.
```
ip a show secondary
```
or
```
ip a show ens5 secondary
```
It looks like we use the `:eip` label on purpose
https://github.com/openshift/sdn/commit/88d43b6d0e8b02a6ece88076d3e1eb28418bb74d#diff-74ce0259328d8a7c7c720e38bd60d2d97df35c3e9cf9acd7fede641fdf4a2b9aR95

```diff
diff --git a/features/networking/egress-ip.feature b/features/networking/egress-ip.feature
index 310c24e5..6449de17 100644
--- a/features/networking/egress-ip.feature
+++ b/features/networking/egress-ip.feature
@@ -140,7 +140,7 @@ Feature: Egress IP related features
     And I wait up to 30 seconds for the steps to pass:
     """
     When I run command on the "<%= node.name%>" node's sdn pod:
-      | bash | -c | ip address show <%= cb.interface %> |
+      | bash | -c | ip address show label <%= cb.interface %>:eip |
     Then the step should succeed
     And evaluation of `@result[:response].chomp.match(/inet.*eip/)[0].match(/\d{1,3}\.\d{1,3}.\d{1,3}.\d{1,3}/)[0]` is stored in the :egress_ip clipboard
     Then the expression should be true> cb.egress_ip == cb.valid_ips[0]
@@ -154,7 +154,7 @@ Feature: Egress IP related features
     And I wait up to 30 seconds for the steps to pass:
     """
     When I run command on the "<%= node.name%>" node's sdn pod:
-      | bash | -c | ip address show <%= cb.interface %> |
+      | bash | -c | ip address show label <%= cb.interface %>:eip |
     Then the step should succeed
     And evaluation of `@result[:response].chomp.match(/inet.*eip/)[0].match(/\d{1,3}\.\d{1,3}.\d{1,3}.\d{1,3}/)[0]` is stored in the :egress_new_ip clipboard
     Then the expression should be true> cb.egress_new_ip == cb.valid_ips[1]
```


```
      [23:40:18] INFO> Exit Status: 0
      [23:40:18] INFO> Shell Commands: oc exec sdn-546bz  --kubeconfig=/home/rbrattai/Documents/deployments/127342/workdir/ocp4_admin.kubeconfig -n openshift-sdn  --container=sdn -i -- bash -c ip\ address\ show\ label\ ens5:eip
          inet 10.0.148.29/19 brd 10.0.159.255 scope global secondary ens5:eip
             valid_lft forever preferred_lft forever
      
```

Reply:
Thanks Ross, updated it!